documentation/src/main/asciidoc/topics/proc_server_configuring_ssl_engine.adoc

```diff
@@ -1,21 +1,28 @@
-[id='ssl_engine-{context}']
-= Tuning SSL Protocols and Cipher Suites
-You can configure the SSL engine, via the {brandname} server SSL identity, to
-use specific protocols and ciphers.
+[id='configuring-ssl-engine_{context}']
+= Configuring TLS versions and cipher suites
+When using SSL/TLS encryption to secure your deployment, you can configure {brandname} Server to use specific versions of the TLS protocol as well as specific cipher suites within the protocol.
 
+.Procedure
+
+. Add the `engine` element to the SSL configuration for {brandname} Server.
+. Configure {brandname} to use one or more TLS versions with the `enabled-protocols` attribute.
++
+{brandname} Server supports TLS version 1.2 and 1.3 by default.
+If appropriate you can remove `TLSv1.2` to restrict the security protocol for client connections to `TLSv1.3`.
+{brandname} does not recommend enabling `TLSv1.1`, however, because it is an older protocol with limited support and offers weak security.
+You should never enable any version of TLS older than 1.1.
```

Comment:
@tristantarrant Could you please give your opinion/blessing on the wording of this TLS recommendation for server? Cheers.